### ISSUE TITLE
When should support the falseCallback also

### ISSUE THREAD

**Issue by ahmad-moussawi** [sqlkata/querybuilder#201]
Currently, it supports only when the condition is true only.

current state:
```cs
query.When(amount  > 0, q => q.Select("revenues"))
          .When(amount <= 0,  q => q.Select("expenses"))
```

suggested
```cs
query.When(amount  > 0, 
   q => q.Select("revenues"),
   q => q.Select("expenses")
)
```

**Comment by ceastwood** [comment]
@ahmad-moussawi  Can we close this one? It looks like your PR #202 resolves this issue for 1.1.7

**Comment by ahmad-moussawi** [comment]
Yes sure I will
